storage/README.md

```diff
@@ -1,5 +1,8 @@
 # Azure Storage SDK for Go
 
+:exclamation: **NOTE:** This Storage Client Library is no longer maintained. 
+Please use the new [Storage Blob SDK for Go](https://github.com/azure/azure-storage-blob-go). New Queue and File SDKs will be available by May of 2018.
```

Comment:
This isn't quite accurate. Only blob support is moving to the new repository. The existing "Storage Client" is still being maintained for files/table/etc. Also in Go, the term "package" is more appropriate than "Library".